csharp/src/Ice/Current.cs

```diff
@@ -21,6 +22,11 @@ public class Current
         /// <summary>The request context, as received from the client.</summary>
         public Dictionary<string, string> Context => IncomingRequestFrame.Context;
 
+        /// <summary>The request deadline. The peer sets the deadline to the absolute time at which its invocation
```

Comment:
What about:

The deadline corresponds to the request's expiration time. Once the deadline is reached, the caller is no longer interested in the response and discards the request. The server-side runtime does not enforce this deadline - it's provided "for information" to the application. The Ice client runtime sets this deadline automatically using the proxy's invocation timeout and sends it with ice2 requests but not with ice1 requests. As a result, the deadline for an ice1 request is always DateTime.MaxValue on the server-side even though the invocation timeout is usually not infinite.

Reply:
Fixed as suggested